client/src/landingPage.js

```diff
@@ -20,6 +20,9 @@ let tagline_options = [
   "put your moisture in the cloud",
   "we want to get you wet... _down there_",
   "for a happy, healthy bush",
+  "fronds with benefits",
+  "sensational plant action",
+  "more growth than you might expect"
```

Comment:
imo, a comma on the last item in a list helps so you can reorder them easily. Note that line 22 had a spurious (but helpful) comma, that meant you didn't need to add it, so you can reorder any item (including the last in the list).

Reply:
I see what you mean, but I feel that manually editing JSON is a last resort, and eventually a UI-driven editor would be best.  Particularly for the taglines, which represent an integral part of the branding of this project.

Reply:
so people should use sensible ide's is what you're saying? agreed!